scrapers/csci_topics_scraper/main.py

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env python3
+
+# Python standard library
+import asyncio
+from operator import itemgetter
+import os
+import re
+import json
+import sys
+
+# External dependnecies
+import aiohttp
+import bs4
+import requests
+
+BeautifulSoup = lambda data: bs4.BeautifulSoup(data, features="lxml")
+
+
+async def get_topics_txts():
+    global session
+    url = "https://www.cs.rpi.edu/~goldsd/docs/"
+    async with session.get(url) as request:
+        soup = BeautifulSoup(await request.text())
+        links = soup.find("table").findAll("a")
+        out = []
+        for link in links:
+            if "topics-courses" in link["href"]:
+                out.append(link["href"])
+        # sorts reverse chronologically
+        return sorted(out, reverse=True, key=lambda x: (x[1:3] + str.lower(x[0])))
+
+
+def goldy_term_to_sis_term(term):
```

Comment:
Can we rename this to goldschmit. I would rather not use a more casual nickname here especially because I don't know if he likes that nickname or not.

Reply:
Yes that's fine